ovh/resource_iploadbalancing_udp_farm_server_gen.go

```diff
@@ -0,0 +1,171 @@
+// Code generated by terraform-plugin-framework-generator DO NOT EDIT.
+
+package ovh
+
+import (
+	"context"
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	ovhtypes "github.com/ovh/terraform-provider-ovh/ovh/types"
+
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+)
+
+func IploadbalancingUdpFarmServerResourceSchema(ctx context.Context) schema.Schema {
+	attrs := map[string]schema.Attribute{
+		"address": schema.StringAttribute{
+			CustomType:          ovhtypes.TfStringType{},
+			Required:            true,
+			Description:         "IPv4 address (e.g., 192.0.2.0)",
+			MarkdownDescription: "IPv4 address (e.g., 192.0.2.0)",
+		},
+		"backend_id": schema.Int64Attribute{
+			CustomType:          ovhtypes.TfInt64Type{},
+			Computed:            true,
+			Description:         "farm id",
```

Comment:
How is it different than `farm_id`? Shoudn't it be `Backend ID` something like that?

Reply:
There are actually the same. cf [doc](https://github.com/ovh/terraform-provider-ovh/pull/719/files#diff-0236891402ed4efdfb119ea8b895aa268f77cbd24634f2ad5d529930ffb4db4eR50)